lib/safira_web/controllers/deliver_redeemable_controller.ex

```diff
@@ -5,6 +5,8 @@ defmodule SafiraWeb.DeliverRedeemableController do
 
   alias Safira.Store
 
+  alias Safira.Contest
```

Comment:
What is this alias used for?

Reply:
It was not needed actually, i forgot to remove it, sorry